lib/src/persist/adapter.dart

```diff
@@ -1,40 +1,41 @@
-import 'package:casbin/src/model/model.dart';
+// Copyright 2018-2021 The Casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Do we need the license on each file @hsluoyz? If so, I would suggest opening a PR exclusively for this change on all files.

Reply:
Yes. Opening a PR for this is good.